src/NzbDrone.Core/MediaCover/MediaCoverService.cs

```diff
@@ -484,21 +484,22 @@ private void DownloadCover(Movie movie, MediaCover cover)
         {
             var fileName = GetMovieCoverPath(movie.Id, cover.CoverType);
 
-            _logger.Info("Downloading {0} for {1} {2}", cover.CoverType, movie, cover.RemoteUrl);
+            _logger.Info("Downloading {0} for {1} {2}", cover.CoverType, movie.ForeignId, cover.RemoteUrl);
```

Comment:
Movie has a ToString that will format the return, the foreignId is within the RemoteUrl, so this is a double up.